easybuild/easyconfigs/b/Biggus/Biggus-0.13.0-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'Biggus'
+version = '0.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/SciTools/biggus/'
+description = """Virtual large arrays and lazy evaluation."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['https://github.com/SciTools/Biggus/archive/']
```

Comment:
now I remember:

    source_urls = [PYPI_SOURCE]
    sources = [SOURCE_TAR_GZ]

didn't work, and I couldn't figure out the download URL for PyPI, since on the PyPI page for biggus, there's no link to the sources.